packages/bigcommerce-payments-integration/README.md

```diff
@@ -0,0 +1,11 @@
+# bigcommerce-payments-integration
+
+This package contains the integration layer for the BigCommerce provider.
```

Comment:
If possible, could you please explain bit more about the purpose of this package?
We have packages like `payment-integration-api` already, `bigcommerce-payments-integration` may confuse developers.

Reply:
Same questions for the rest 2 related PRs.
We may need to align better on the names of the three packages.

Reply:
bigcommerce-payments is a new payment provider, so based on our rules for package creation (and long discussions with payments core team), we decided to use this naming.

Reply:
I think in this case it might we can thing about changing `payment-integration-api` package name to `integration-api` since it contains logic that may be used in different (non payment integrations) packages